src/main/java/com/matchday/matchdayserver/common/response/MatchUserStatus.java

```diff
@@ -0,0 +1,31 @@
+package com.matchday.matchdayserver.common.response;
+
+public enum MatchUserStatus implements StatusInterface {
+    NOTFOUND_MATCH(400, 7001, "존재하지 않는 매치입니다"),
+    NOTFOUND_USER(400, 7002, "존재하지 않는 유저입니다");
```

Comment:
```suggestion
    NOTFOUND_MATCH(404, 7001, "존재하지 않는 매치입니다"),
```

http status code는 not found니까 404가 어떨까요

그리고 not found user는 UserStatus에 있는거 써도 될 것 같습니다